packages/core/src/Month.js

```diff
@@ -572,7 +572,7 @@ export class Month extends TemporalAccessor {
             return Month.of(temporal.get(ChronoField.MONTH_OF_YEAR));
         } catch (ex) {
             throw new DateTimeException('Unable to obtain Month from TemporalAccessor: ' +
-                    temporal + ' of type ' + (temporal && temporal.constructor != null ? temporal.constructor.name : ''), ex);
+                    String(temporal) + ' of type ' + (temporal && temporal.constructor != null ? temporal.constructor.name : ''), ex);
```

Comment:
And why using here String ? Nowadays i would prefer template literals. WDYT ?

Reply:
In general I also prefer template literals. In the changes I made I kept the code as it was (except in tests) just adding the necessary parts to keeping it working. Here I used `String` instead of `.toString()` because this method is called with `null` in a test and `String()` works with `null` and `undefined` while `.toString()` doesn't.

Do you think is better to convert this to templates literals instead?

Reply:
Not sure if i get it, isnt it the same as ``${temporal} of type ${temporal && temporal.constructor != null ? temporal.constructor.name : ''}``

Reply:
Yes, changing to a template literal is clearer. I just "fix" the tests by putting a bunch of `.toString()` where was needed. I couldn't use `.toString()` in this particular case because there is a test that calls this function with `null` and can't call `.toString()` with `null`, causing the test to fail. Using `String()` instead was the easy fix.